docs/applications/homebrew.md

```diff
@@ -3,17 +3,48 @@ id: homebrew
 title: Homebrew
 ---
 
-
 ## About
+
 > Homebrew is a free and open-source software package management system that simplifies the installation of software on Apple's macOS operating system and Linux
-> [(Wikipedia, 2020)](https://en.wikipedia.org/wiki/Homebrew_(package_manager))
+> [(Wikipedia, 2020)](<https://en.wikipedia.org/wiki/Homebrew_(package_manager)>)
```

Comment:
```suggestion
> [(Wikipedia, 2020)](https://en.wikipedia.org/wiki/Homebrew_(package_manager))
```
Since we are already using `[]()` the `<>`-brackets are not needed.